app/views/TaxYearSummary.scala.html

```diff
@@ -231,6 +246,9 @@ <h2 class="govuk-heading-m">@messages(header)</h2>
                             @documentDetailWithDueDate.documentDetail.getBalancingChargeDueDate(codingOutEnabled).map(_.toLongDateShort).getOrElse(messages("tax-year-summary.na"))
                          </td>
                          <td class="govuk-table__cell govuk-table__cell--numeric">@documentDetailWithDueDate.documentDetail.originalAmount.toCurrencyString</td>
+                    } else if(documentDetailWithDueDate.isReviewAndReconcileDebit) {
+                         <td class="govuk-table__cell">@documentDetailWithDueDate.dueDate.map(_.toLongDateShort).getOrElse(messages("tax-year-summary.na"))</td>
```

Comment:
we need to double check this with BA, as setting default value is not part of the requirements